src/components/pages/AddFarmer/AddFarmer.jsx

```diff
@@ -0,0 +1,160 @@
+/** @format */
+
+import React, {useState} from 'react'
+import Input from '../../common/Input/Input'
+import {personalInfo, familyInfo, guarantor, farmInfo} from '../../common/Input/addFarmerData'
+import Button from '../../common/Button/StyledButton'
+import styled from 'styled-components'
+import axios from 'axios'
+
+const AddFarmer = () => {
+  const [state, setState] = useState({
+    personalInfo: personalInfo,
+    familyInfo: familyInfo,
+    guarantor: guarantor,
+    farmInfo: farmInfo,
+  })
+  const [stateToggle, setStateToggle] = useState({
+    personalInfoToggle: false,
+    familyInfoToggle: true,
+    guarantorToggle: true,
+    farmInfoToggle: true,
+  })
+
+  const onChangeHandler = (e, data) => {
+    const {name, value, type} = e.target
+    const newData = {...state[data]}
+    const newEntry = {...newData[name]}
+    if (type === 'checkbox') {
+      if (newEntry.selected.indexOf(value) > -1) {
+        newEntry.selected = newEntry.selected.filter(s => s !== value)
+      } else {
+        newEntry.selected = [...newEntry.selected, value]
+      }
+    } else {
+      newEntry.value = value
+    }
+    newData[name] = newEntry
+    setState({...state, [data]: newData})
+  }
+  const toggleHandler = data => {
+    setStateToggle(prevState => ({
+      ...prevState,
+      [data]: !prevState[data],
+    }))
+  }
+  const formHandler = e => {
+    e.preventDefault()
+    let formData = {}
+
+    const newState = JSON.parse(JSON.stringify(state))
+    for (let key in newState) {
+      formData[key] = newState[key]
+      for (let key2 in newState[key]) {
+        if (newState[key][key2].selected) {
+          formData[key][key2] = newState[key][key2].selected
+        } else {
+          formData[key][key2] = newState[key][key2].value
+        }
+      }
+    }
+    e.target.reset()
+    const headers = {
+      'Content-Type': 'application/json',
+      Authorization: localStorage.getItem('tokenTiemeNdo'),
+    }
+    axios
```

Comment:
Please put your axios request into the appropriate handler file and imprt from there